packages/db-ui-elements-stencil/tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "jsx": "react",
     "jsxFactory": "h",
     "esModuleInterop": true,
-    "allowJs": true
+    "allowJs": true,
+    "skipLibCheck": true
```

Comment:
@nmerget what's the reason for this configuration?

Reply:
https://www.typescriptlang.org/tsconfig#skipLibCheck

There was an issue with @types/jest using some older types from node<16

Reply:
@nmerget when does this problem come up, like e.g. during which npm script?